users/tuning.rst

```diff
@@ -0,0 +1,192 @@
+    large periodic scans. Changes are detected faster using notifications
+    than scans.
+
+- :opt:`copiers`
+    The number of routines used for the copy stage of file syncing. Similar
+    to other concurrency options, if there are a lot of files to sync and if
+    the I/O system can handle it it you may see increased performance by
+    increasing this value. The default is system dependent, somewhere
+    between 1 and the number of CPU cores available.
+
+- :opt:`hashers`
+    When hashing locally changed files, the number of hashing routines to
+    use. Higher values mean more I/O and CPU load and may increase
+    performance when there are a lot of files to hash, assuming I/O
+    bandwidth and CPU are available. The default is system dependent,
+    somewhere between 1 and the number of CPU cores available.
+
+- :opt:`pullerMaxPendingKiB`
+    The maximum amount of data to have outstanding requests for at any given
+    time. Higher values may improve performance, especially if the network
+    or I/O latency is high. The default is 32 MiB.
+
+- :opt:`scanProgressIntervalS`
+    Providing the GUI with scan progress updates is not very expensive, but
+    is effort that could be spent on something more productive. If you don't
+    need to see scan progress, set this to -1 to disable it.
+
+- :opt:`weakHashThresholdPct`
+    Syncthing will by default look for rolling (weak) hash matches to detect
+    data shifted in a file if a lot of data has changed in the file. If your
+    use case doesn't cause data to shift in a file, and if the files are
+    large (movies, VM images, ...) it is unnecessary to spend time checking
+    weak hashes. Set the threshold to 101% to disable use of weak hashes.
+
+- :opt:`maxConcurrentWrites`
+    Synchting limits the number of outstanding write system calls at any
+    given time to avoid overloading the I/O system. If you increased
+    copiers, outstanding network requests, or other settings that increase
+    the number of concurrent writes, you may need to increase this value.
+    The default is 2.
+
+- :opt:`disableFsync`
+    Syncthing calls ``fsync()`` on files and directories after syncing them
+    to ensure they are safe and sound on stable storage. This is a good
+    thing, but it can be expensive. If you have a lot of files to sync
+    and/or a lot of I/O bandwidth available, you may see a performance
+    increase by disabling ``fsync()``. This is not recommended for most
+    setups, as you are increasing the risk of data loss in case of a power
+    outage or system crash.
+
+- :opt:`blockPullOrder`
+    Syncthing by default uses a pseudo-random block order when pulling in
+    order to distribute load over multiple devices better. If you are
+    generally downloading files from only one device, and if you have
+    spinning disks as the underlying storage, you may see a performance
+    increase by setting this to ``inOrder``.
+
+- :opt:`copyRangeMethod`
```

Comment:
This should also go in the low end section as copyrange uses less memory and cpu in syncthing